rhythm.py

```diff
@@ -232,7 +237,10 @@ def on_bell_ring(self, bell, stroke, real_time):
         # If this bell was expected at this stroke (i.e. is being rung by someone else)
         if (bell, stroke) in self._expected_bells:
             # Figure out where the bell was expected in ringing space
-            expected_blow_time = self._expected_bells[(bell, stroke)]
+            (row_number, place) = self._expected_bells[(bell, stroke)]
```

Comment:
Now calculating when the expected time before each bell instead of at the start of the row? Does this make it more responsive initially?

Reply:
This particular change is just a side effect of me making sure that the inertia is 0 when pulling off.  In order to do this, the bell-after-expected callback has to know the row index, which means that it instead of precalculating `expected_blow_time` (which doesn't depend on the regression line), it stores the two components `row_number` and `place` so that they can be combined later but still have the option to use them individually.